streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -331,25 +340,29 @@
                                                                 Serde<T> aggValueSerde);
 
     /**
-     * Aggregate values of this stream by key on a window basis.
+     * Aggregate values of this stream by key on a window basis into a new instance of windowed {@link KTable}
```

Comment:
Typo: "into a new instance of [a] windowed {@link KTable}."

Also: Full stop `.` missing at the end.